src/controllers/eventController.js

```diff
@@ -21,9 +22,9 @@ eventController.post('/register', async (req, res) => {
         if (Object.keys(req.body).length === 0) {
             throw new Error('Invalid request Body!')
         }
-        if (req.body.dates[0].startTime >= req.body.dates[0].endTime) {
-            throw new Error('Start-time can\'t be after or equal to end-time!')
-        }
+        // TODO: Ask can we create event with empty date array?
+
+        duplicateDates(req.body.dates, req.body.dates[0]?.date);
```

Comment:
what if the event has 3 dates and the 2nd and the 3rd are duplicated?

Reply:
Hristo fixed this already. The organizer can't set two same dates.